guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -340,40 +349,71 @@ export default class AudioPlayerComponent extends Component {
   play() {
     // TODO
   }
-}
-```
-
-We'd like to call the `play` method on our `<audio>` element, but how do we get access to that element?
-
-We can give our component access to elements inside of it by using [`ember-ref-modifier`](https://github.com/lifeart/ember-ref-modifier).
-
-> The `{{ref}}` modifier takes a property name and an object, and assigns the modifier's element to that property on the object.
-
-In this case, we'll assign the `<audio>` element to the `audioElement` property in our component:
-
-```handlebars {data-filename="app/components/audio-player.hbs" data-diff="-1,+2"}
-<audio src={{@srcURL}} />
-<audio src={{@srcURL}} {{ref this "audioElement"}} />
 
-<button type="button" {{on "click" this.play}}>Play</button>
-<button type="button">Pause</button>
+  @action
+  pause() {
+    // TODO
+  }
+}
 ```
 
-Now, the component can access the audio element in the `play` method:
+From the component's point of view, all it needs to do is track whether it should be playing or not, since we're going to manage the DOM element with the modifier. So we can just use `@tracked` and a normal property here.
 
-```js {data-filename="app/components/audio-player.js" data-diff="-7,+8"}
+```js {data-filename="app/components/audio-player.js" data-diff="+2,+6,+7,-10,+11,-16,+17"}
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
 import { action } from "@ember/object";
 
 export default class AudioPlayerComponent extends Component {
+  @tracked isPlaying = false;
+
   @action
   play() {
     // TODO
-    this.audioElement.play();
+    this.isPlaying = true;
+  }
+
+  @action
+  pause() {
+    // TODO
+    this.isPlaying = false;
   }
 }
 ```
 
+Now the component is managing the state of the player, but we still need to translate that into the DOM method calls. So let's create a modifier!
```

Comment:
```suggestion
Notice that the component manages the state of the audio player. We still need to translate the state into the appropriate DOM method call. Let's create the modifier now!
```

Reply:
Did a slight variant on this in b1da6ccd!